.travis.yml

```diff
@@ -29,50 +29,49 @@ cache:
 
 jobs:
   include:
-    - stage: "Build front and upload docker image"
-      before_script:
+    - stage: "Build the frontend and upload a Docker image"
+      script:
         - cd front
         - npm install
         - npm run build
         - cd -
-      script:
-        - export BRANCH="$(echo "$TRAVIS_BRANCH" | tr '/' '-')"
-        - make front/travis-docker "tag=$BRANCH--front"
-        - docker login quay.io -u "$DOCKER_USER" -p "$DOCKER_PASS"
-        - docker push "quay.io/aelve/guide:$BRANCH--front"
-      if: type = push
```

Comment:
I changed this to a bash `if` to make it more clear that the stage should run for both PRs and pushes, and only the Docker part should run for pushes.

Reply:
:+1: